tmt/base.py

```diff
@@ -3338,6 +3338,8 @@ def finish(self) -> None:
             raise SystemExit(1)
         if stats[ResultOutcome.PASS]:
             raise SystemExit(0)
+        if stats[ResultOutcome.SKIP]:
+            raise SystemExit(2)
```

Comment:
I'd rather suggest to make this aligned with how `pytest` handles skipped tests:

```python
import pytest

@pytest.mark.skip
def test_something():
    assert False
```

Which seems logical:

```
===================================== test session starts ======================================
platform linux -- Python 3.11.6, pytest-7.1.3, pluggy-1.0.0 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /tmp/pytest
plugins: anyio-3.5.0
collected 1 item                                                                               

test_ha.py::test_something SKIPPED (unconditional skip)                                  [100%]

====================================== 1 skipped in 0.00s ======================================
psss@fedora:/tmp/pytest> echo $?
0
```

Reply:
Your change would change the exit code, that was not my goal here. I wanted to make `SKIP` pop up in the code, to make it clear how it’s mapped to an exit code.

We felt the actual change of the exit code is worth the discussion, therefore it was put on tomorrow agenda. There might be a patch changing it, or we’d stick to the current one, who knows. Here I wanted only to change implicit to explicit, which I felt should happen in either case - a follow-up patch may change the code, but the explicit branch fir `SKIP` would remain.

Reply:
> We felt the actual change of the exit code is worth the discussion, therefore it was put on tomorrow agenda.

Ack, let's agree on the approach on the hacking session. I just don't see benefit in two separate patches here. Would suggest to implement the decision right away here.

Reply:
I believe this was resolved, we decided to go with exit code 4